### PR TITLE
Options module

### PR DESCRIPTION
**Pull request by SuselMan**

### CODE REVIEW

File: app/css/style.css [hunk 1]
```diff
@@ -107,3 +147,14 @@ body {
...
   position: absolute; }
+
+.popup-contanier {
+  z-index: 5000;
+  position: absolute;
+  top: 0;
+  right: 0; }
+
+.popup {
+  -webkit-box-shadow: 0px 0px 33px -5px rgba(0, 0, 0, 0.12);
```
Comment (TanyaGaleyev) [inline app/css/style.css:158]: sass cannot do it in a smart way to avoid 3 times duplication?

File: app/index.html [hunk 2]
```diff
@@ -25,39 +25,46 @@
 
 <body class="color-default bg-light-dark theme-sltl theme-sdtl">
 
+<div id="loading-indicator">
+    <img src="images/ajax-loader.gif">
```
Comment (TanyaGaleyev) [inline app/index.html:29]: should img tag be closed?

File: app/js/app.js [hunk 3]
```diff
@@ -43,7 +48,8 @@ define(function (require) {
 	});
 
 	channelGlobal.reply('hide:app:loader', function () {
-		$('.loader-overlay').addClass('loaded');
+		//$('.loader-overlay').addClass('loaded');
```
Comment (TanyaGaleyev) [inline app/js/app.js]: may be just delete this line?

File: app/js/common/modelbinder_converters.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+define(function (require) {
+	"use strict";
+
+
+	var Converters = {
+		CB: function (direction, value) {
```
Comment (TanyaGaleyev) [inline app/js/common/modelbinder_converters.js:6]: what does CB stands for?

File: app/js/modules/midi/midi.js [hunk 5]
```diff
@@ -9,17 +9,14 @@ define(function(require){
...
     var Midi=function(){
-        if (!is) {
-            obj=new MidiWacher();
-        } else {
-            console.warn("уже создан");
-        };
+        if(!is){obj=new MidiWacher()};
         is=true;
-        return obj;
+        return obj
```
Comment (TanyaGaleyev) [inline app/js/modules/midi/midi.js]: semicolon?

File: app/js/modules/midi/midi.js [hunk 6]
```diff
@@ -74,33 +66,59 @@ define(function(require){
...
-                if(e.data[2]>0){
-                    channelMidi.trigger("note:on",0, e.data[1]);
-                }
-                else{
-                    channelMidi.trigger("note:off",0, e.data[1]);
-                }
+        this.access=access;
+        setting.inputs=[];
+        setting.outputs=[];
+        access.inputs.forEach( function( port, key ) {
```
Comment (TanyaGaleyev) [inline app/js/modules/midi/midi.js:70]: Is for each same in this context. I read something that in some cases for with indexes is safer.

File: app/js/modules/midi/midi.js [hunk 7]
```diff
@@ -74,33 +66,59 @@ define(function(require){
...
+            var device={};
+            console.log("PORT",port);
+            device.text = port.name;
+            device.id=port.id;
+            setting.inputs.push(device)
```
Comment (TanyaGaleyev) [inline app/js/modules/midi/midi.js]: mb inline?

``` js
settings.inputs.push({
  text: port.name,
  id: port.id
});
```

and also semicolon

File: app/js/modules/midi/midi.js [hunk 8]
```diff
@@ -74,33 +66,59 @@ define(function(require){
...
+        });
+        access.outputs.forEach( function( port, key ) {
+            var device={};
```
Comment (TanyaGaleyev) [inline app/js/modules/midi/midi.js]: inline?

File: app/js/modules/midi/midi.js [hunk 9]
```diff
@@ -108,6 +126,7 @@ define(function(require){
 
     });
 
-    return Midi;
+    return Midi
```
Comment (TanyaGaleyev) [inline app/js/modules/midi/midi.js:126]: semicolon?

File: app/js/modules/midi/midi.js [hunk 10]
```diff
@@ -108,6 +126,7 @@ define(function(require){
...
+
 
-});
+})
```
Comment (TanyaGaleyev) [inline app/js/modules/midi/midi.js:129]: semicolon?

File: app/js/modules/settings/model.js [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+ * Created by i.pavluhin on 26.10.2015.
+ */
+
+define(function (require) {
+    "use strict";
+
+    var Marionette = require('marionette'),
+        Backbone = require('backbone');
+    var $=require('jquery');
+    var Radio = require('backbone.radio');
```
Comment (TanyaGaleyev) [inline app/js/modules/settings/model.js]: usuned _requires_

File: app/js/modules/pianoroll/view.js [hunk 12]
```diff
@@ -67,6 +67,11 @@ define(function (require) {
 
             }.bind(this));
 
+
```
Comment (TanyaGaleyev) [inline app/js/modules/pianoroll/view.js:70]: ???

File: app/js/appController.js [hunk 13]
```diff
@@ -8,6 +8,8 @@ define(function(require) {
   var App = require('app');
   var channelGlobal = require('backbone.radio').channel('global');
 
+  require('modules/settings/module');
```
Comment (stytz) [inline app/js/appController.js]: Is it necessary? Seems redundant, as before. You have _require_ below.

**Pull request merged**